.coveragerc

```diff
@@ -10,3 +10,4 @@ omit =
     */python?.?/*
     */site-packages/nose/*
     */test/*
+    */constraint.py
```

Comment:
i'd say to not exclude constraint.py here.. just let it die a natural death.  this has the potential to get left behind, masking any future constraint module